include/net/socket_ncs.h

```diff
@@ -16,18 +16,6 @@
 extern "C" {
 #endif
 
-/* NCS specific protocol/address families. */
-
-#define PF_LTE 102 /**< Protocol family specific to LTE. */
-#define AF_LTE PF_LTE /**< Address family specific to LTE. */
```

Comment:
There is one occurrence of `AF_LTE` and `NPROTO_AT` in `nrf/lib/at_cmd/at_cmd.c` that I don't see in the sdk-nrf PR [[PR-36]](https://github.com/lemrey/sdk-nrf/pull/36/files) Is anyone working on rewriting/removing this?

Reply:
That library will be removed before we release `v1.5.0`